web_src/js/features/common-global.js

```diff
@@ -35,15 +35,18 @@ export function initHeadNavbarContentToggle() {
   });
 }
 
-export function initFootLanguageMenu() {
-  function linkLanguageAction() {
-    const $this = $(this);
-    $.get($this.data('url')).always(() => {
+export function initFooterMenus() {
+  $('.language-menu a[lang]').on('click', function() {
+    $.get($(this).data('url')).always(() => {
```

Comment:
Isn't this a `GET`?

Reply:
Yes, could be refactored to use the new fetch, I touched it only to get rid of the previous variable.